src/Apps/__stories__/Collect.story.tsx

```diff
@@ -32,9 +32,6 @@ storiesOf("Apps/Collect/Collection", module)
   })
   .add("Collection with Hub rails", () => {
     return (
-      <MockRouter
-        routes={collectRoutes}
-        initialRoute="/collection/contemporary"
```

Comment:
Had to change the sample route cuz I couldn't look at the work of a certain someone. 😄